src/UI/Components/ServiceInstanceForm/Components/AutoCompleteInputProvider.tsx

```diff
@@ -67,10 +83,13 @@ export const AutoCompleteInputProvider: React.FC<Props> = ({
               ? service_identity_attribute_value
               : id;
 
+            const isSelected =
+              alreadySelected !== null && alreadySelected.includes(id); //it can be that the value for this inter-serviceRelation is set to null
```

Comment:
```suggestion
              alreadySelected !== null && alreadySelected.includes(id); //it can be that the value for inter-service relation is set to null
```